### PR TITLE
TIO sees status for payment request in table and single item view

### PR DESCRIPTION
**Pull request by SirenaBorracha**
## Description

TIO can now see the status of a payment request in both table and detail view.

## Setup

```sh
make server_run
make office_client_run
```
navigate to http://officelocal:3000/payment_requests

## Code Review Verification Steps
* [ ] There are no aXe warnings for UI.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-692) for this change

## Screenshots
Table View
Screen Shot 2020-01-06 at 1 46 34 PM

Detail View
Screen Shot 2020-01-06 at 1 05 09 PM

### CODE REVIEW

File: pkg/assets/assets.go [hunk 1]
```diff
@@ -453,20 +453,33 @@ func AssetNames() []string {
```
Comment (donaldthai) [inline pkg/assets/assets.go]: Can you exclude this file in this PR? There's a new version for `go-bindata` that we should update separately.

File: src/scenes/Office/TIO/paymentRequestIndex.jsx [hunk 2]
```diff
@@ -31,6 +32,7 @@ class PaymentRequestIndex extends React.Component {
                 <td>{`${pr.isFinal}`}</td>
                 <td>{pr.rejectionReason}</td>
                 <td>{pr.serviceItemIDs}</td>
+                <td>{pr.status}</td>
```
Comment (donaldthai) [inline src/scenes/Office/TIO/paymentRequestIndex.jsx]: Looks like we need the `key` attribute to the `<tr>` element. 
image

Comment (sarboc) [reply src/scenes/Office/TIO/paymentRequestIndex.jsx]: This article had some interesting info on why keys are used and the best practices for them: https://reactjs.org/docs/lists-and-keys.html. tl;dr the code you added above, @SirenaBorracha, looks great!

File: swagger/prime.yaml [hunk 3]
```diff
@@ -720,9 +720,11 @@ definitions:
...
   PaymentRequestStatus:
     enum:
-      - PAYMENT_SUBMITTED
-      - APPROVED
-      - REJECTED
+      - PENDING
+      - REVIEWED
+      - SENT_TO_GEX
+      - RECEIVED_BY_GEX
+      - PAID
```
Comment (kimallen) [inline swagger/prime.yaml]: Thanks for updating this in the `prime.yaml` too!

File: pkg/assets/assets.go (hunk 1 above)
Comment (SirenaBorracha) [reply pkg/assets/assets.go]: Done!

**Pull request merged**